src/DynamoCore/Graph/Workspaces/WorkspaceInfo.cs

```diff
@@ -49,6 +49,7 @@ public WorkspaceInfo()
                 double cx = 0;
                 double cy = 0;
                 double zoom = 1.0;
+                double scaleFactor = 1.0;
```

Comment:
thanks @aparajit-pratap - good to know, first question that comes to mind which is not obvious to me from this code is where does the `scalefactor` get injected into LibG?